src/components/Team/UpdateProfile.js

```diff
@@ -24,22 +24,13 @@ const UploadContainer = styled.div`
 
 const UpdateProfile = props => {
 	const [user, setUser] = useState(null);
-	const [name, setName] = useState(null);
 	const [isLoading, setIsLoading] = useState(false);
-	const [email, setEmail] = useState(null);
-	const [password, setPassword] = useState(null);
+	const [newName, setNewName] = useState(null);
```

Comment:
These are not required in the component state. Name and email changes are already being handled on the backend.